subsys/logging/Kconfig

```diff
@@ -279,6 +280,58 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FS
+	bool "Enable LittleFS backend"
+	depends on FILE_SYSTEM_LITTLEFS
+	default n
+	help
+	  When enabled backend is using LittleFS to output logs.
+
+if LOG_BACKEND_FS
+
+config LOG_BACKEND_FS_AUTOSTART
+	bool "Enable file system backend start automatically"
+	default y
```

Comment:
IIRC there was some other backend (net?) that recently had to be made not auto-start.  Is this safe when the file system is on an external flash device that isn't available until late `POST_KERNEL`?